app/controllers/submissions_controller.rb

```diff
@@ -1,11 +1,16 @@
 class SubmissionsController < ApplicationController
   def new
     @submission = Submission.new
+    @submission.submission_packets.build.build_poem
```

Comment:
This _technically_ violates demeter. It's not necessarily a problem, but it might be worth wrapping in a method on `Submission`.